api/types.proto

```diff
@@ -776,3 +776,23 @@ message SecretReference {
 	// lookup/display purposes.  The secret in the reference will be identified by its ID.
 	string secret_name = 4;
 }
+
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+message HealthConfig {
+	// Test is the test to perform to check that the container is healthy.
+	// An empty slice means to inherit the default.
+	// The options are:
+	// {} : inherit healthcheck
+	// {"NONE"} : disable healthcheck
+	// {"CMD", args...} : exec arguments directly
+	// {"CMD-SHELL", command} : run command with system's default shell
+	repeated string test = 1;
+
+	// Zero means to inherit. Durations are expressed as integer nanoseconds.
```

Comment:
This comment is incorrect. These are Duration messages, which is not interesting for this particular structure.